packages/rum-core/src/domain/configuration.ts

```diff
@@ -144,6 +146,109 @@ export function validateAndBuildRumConfiguration(
   )
 }
 
+/**
+ * Handles configureTracingUrls and processes legacy allowedTracingOrigins
+ */
+function handleTracingParameters(initConfiguration: RumInitConfiguration): ConfigureTracingOption[] | undefined {
+  // Advise about parameters precedence.
+  if (initConfiguration.configureTracingUrls !== undefined && initConfiguration.allowedTracingOrigins !== undefined) {
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏it might be a bit late to talk about wording, but seeing all those `configuration.configure` makes me tick: all other configuration values also configure something, not this one in particular, so I find it weird to explicitly mention `configure` here. Maybe we could think of a better name for `configureTracingUrls`. Maybe `tracingUrls` is enough? WDYT?